tests/perf/test_accuracy.py

```diff
@@ -0,0 +1,143 @@
+        ("tests/assets/cheetah_voc", None),
+    ],
+)
+class TestAccuracy:
+    MODEL_NAME = "mlc_mobilenetv3_large_voc"
+
+    @pytest.fixture(autouse=True)
+    def setup(self, fxt_data_root, data_root, ann_path):
+        data_dir = fxt_data_root
+        retrieve_otx_model(data_dir, self.MODEL_NAME)
+        model_path = data_dir / "otx_models" / (self.MODEL_NAME + ".xml")
+        model = ov.Core().read_model(model_path)
+
+        self.setup_dataset(data_root, ann_path)
+
+        self.preprocess_fn = get_preprocess_fn(
+            change_channel_order=self.channel_format == "BGR",
+            input_size=(224, 224),
+            hwc_to_chw=True,
+        )
+        self.postprocess_fn = get_postprocess_fn(activation=ActivationType.SIGMOID)
+
+        self.explainer = Explainer(
+            model=model,
+            task=Task.CLASSIFICATION,
+            preprocess_fn=self.preprocess_fn,
+            explain_mode=ExplainMode.WHITEBOX,
+        )
+
+        self.pointing_game = PointingGame()
+        self.auc = InsertionDeletionAUC(model, self.preprocess_fn, self.postprocess_fn)
+        self.adcc = ADCC(model, self.preprocess_fn, self.postprocess_fn, self.explainer)
+
+    def setup_dataset(self, data_root: str, ann_path: str):
+        self.dataset_type = define_dataset_type(data_root, ann_path)
+        self.channel_format = "RGB" if self.dataset_type in [DatasetType.VOC, DatasetType.COCO] else "None"
+
+        if self.dataset_type == DatasetType.COCO:
+            self.dataset = datasets.CocoDetection(root=data_root, annFile=ann_path)
+            self.dataset_labels_dict = {cats["id"]: cats["name"] for cats in self.dataset.coco.cats.values()}
+            self.anns_to_gt_bboxes = coco_anns_to_gt_bboxes
+        elif self.dataset_type == DatasetType.VOC:
+            self.dataset = datasets.VOCDetection(root=data_root, download=False, year="2012", image_set="val")
+            self.dataset_labels_dict = None
+            self.anns_to_gt_bboxes = voc_anns_to_gt_bboxes
+
+    def test_explainer_images(self):
+        images, explanations, dataset_gt_bboxes = [], [], []
+        for image, anns in self.dataset:
+            image_np = np.array(image)
+            gt_bbox_dict = self.anns_to_gt_bboxes(anns, self.dataset_labels_dict)
+            targets = [target for target in gt_bbox_dict.keys() if target in VOC_NAMES]
+
+            explanation = self.explainer(image_np, targets=targets, label_names=VOC_NAMES, colormap=False)
+
+            images.append(image_np)
+            explanations.append(explanation)
+            dataset_gt_bboxes.append({key: value for key, value in gt_bbox_dict.items() if key in targets})
+
+        pointing_game = self.pointing_game.evaluate(explanations, dataset_gt_bboxes)
+        auc = self.auc.evaluate(explanations, images, steps=10)
+        adcc = self.adcc.evaluate(explanations, images)
+
+        return {**pointing_game, **auc, **adcc}
```

Comment:
What does this "return" mean?

Reply:
It's a temporary approach before finalizing performance tests to check that datasets were processed and all metrics were calculated without any issues.